treeherder/intermittents_commenter/management/commands/run_intermittents_commenter.py

```diff
@@ -1,16 +1,27 @@
+import calendar
+from datetime import date
+
 from django.core.management.base import BaseCommand
 
 from treeherder.intermittents_commenter.commenter import Commenter
 
 
 class Command(BaseCommand):
     """Management command to manually initiate intermittent failures commenter
-       default is daily and non-test mode, use flags for weekly and test mode."""
+       default is daily and non-test mode, use flags for weekly, auto and test mode."""
 
     def add_arguments(self, parser):
-        parser.add_argument('-w', '--weekly', action='store_true', dest='weekly_mode', help='generate weekly, rather than daily, comment summaries')
+        parser.add_argument('-m', '--mode', dest='weekly_mode', nargs='?', choices=['weekly', 'auto'], default=False, help='generate comment summaries based on auto or weekly mode')
```

Comment:
To make the output of `--help` for this command clearer, what do you think about including `'daily'` in the choices, and making it the default value? (I don't expect anyone will actually pass `--mode daily` but instead omit `--mode` entirely)

Reply:
So making `default='daily'`rather than `False`? Or are you saying to include daily as a choice yet still default to False? This feels a bit redundant. I could instead change the `help` message to note that the default is 'daily'. 

If you want daily included as a choice, maybe it's better to change `nargs=+` so it shows an error message if an arg isn't passed in after `--mode`?

Reply:
> I could instead change the `help` message to note that the default is 'daily'.

happy for you to do this instead :-)